code/modules/events/abductor.dm

```diff
@@ -0,0 +1,61 @@
+/datum/round_event_control/abductor
+	name = "Abductors"
+	typepath = /datum/round_event/abductor
+	weight = 5
+	max_occurrences = 1
+
+	earliest_start = 180000 // 30 min
+
+	gamemode_blacklist = list("nuclear","wizard","cult","gang","revolution","abduction","blob")
```

Comment:
@Incoming5643 
>the only reason this var existed was to keep random blobs out of blob mode.

umm, this var was added for this PR, so unless you were there for some conversation with Anturk I wasn't, you're assuming his intentions, which isn't great.